Apps/Sandcastle/gallery/Particle System Weather.html

```diff
@@ -95,42 +91,20 @@
             particle.endColor.alpha = 0.0;
           } else {
             particle.endColor.alpha =
-              snowSystem.endColor.alpha / (distance / snowRadius + 0.1);
+              Cesium.Color.WHITE.withAlpha(1.0).alpha /
```

Comment:
This will always evaluate to `1.0`, so let's simplify:
```suggestion
              1.0 /
```